streams/block/ciphers/misty1/misty1.h

```diff
@@ -0,0 +1,41 @@
+/**
+ * Source: https://botan.randombit.net/doxygen/dir_c85d3e3c5052e9ad9ce18c6863244a25.html
```

Comment:
I have not found it by this link. What about: https://github.com/randombit/botan/tree/release-2/src/lib/block/misty1

(intentionally fixed to specific release, in future, they will probably remove MISTY from Botan).